### PR TITLE
User reg auth

### PR DESCRIPTION
**Pull request by Akasora39**

### CODE REVIEW

**Comment by synzen** [comment]
looks like you may have missed the path changes in the merge (`client/src/js/components` rather than `client/src/components`) for where components go (also the imports in App.js)

also the server dependencies were installed in the root package.json rather than [server/package.json](https://github.com/CSC59939/QuickBorrow/blob/master/server/package.json) - not sure if this is what you intended

File: package.json [hunk 1]
```diff
@@ -6,12 +6,25 @@
...
+    "bcryptjs": "^2.4.3",
+    "body-parser": "^1.19.0",
+    "express": "^4.17.1",
+    "is-empty": "^1.2.0",
+    "jsonwebtoken": "^8.5.1",
+    "mongoose": "^5.7.7",
+    "passport": "^0.4.0",
+    "passport-jwt": "^4.0.0",
+    "validator": "^11.1.0"
+  },
```
Comment (yliu0715) [inline package.json]: server dependencies should be in the server directory and client dependencies should be in client directory

File: client/src/js/components/App.js [hunk 2]
```diff
@@ -2,6 +2,16 @@ import React from 'react';
...
+<<<<<<< HEAD
+import Homepage from './Homepage/Homepage';
+import Register from './Register/Register';
+import Login from './Login/Login';
+=======
+import Homepage from './components/Homepage/Homepage';
+import Register from './components/Register/Register';
+import Login from './components/Login/Login';
+>>>>>>> 443c33d4f839c5178919d3de0b6ed13bc591e5de
+
```
Comment (yliu0715) [inline client/src/js/components/App.js]: Did you create a folder for each individual component? I think you might want the lower part here

Comment (Akasora39) [reply client/src/js/components/App.js]: Sorry about that, merging issue that I forgot to catch after pushing.

**Pull request closed without merging**